roles/st2/tasks/main.yml

```diff
@@ -18,12 +18,25 @@
     - reload st2
   tags: st2, skip_ansible_lint
 
+- name: Install present st2 package, no auto-update
+  become: yes
+  package:
+    name: st2
+    state: present
+  when: st2_version == "present"
+  notify:
+    - restart st2
+    - reload st2
+  tags: st2
+
 - name: Install pinned st2 package
   become: yes
   package:
     name: st2={{ st2_version }}-{{ st2_revision }}
```

Comment:
After check, needs an adjustement, since it's `apt` format: `apt install st2=2.2.1-5`
yum is different: `yum install st2-2.2.1-5`.

It's interesting that revision is not a requirement for yum. It's possible to do: `yum install st2-2.2.1`, while it's not possible to specify version without revision in apt.